### PR TITLE
feat(docs): Add more information to the header and footer of the README

### PR DESCRIPTION
**Pull request by nderjung**
This commit includes a GIF preview of KraftKit's `kraft` program in action and  provides some additional links to features and information of KraftKit and the Unikraft OSS Project.

### Prerequisite checklist

  - [x] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
  - [x] Tested your changes against relevant architectures and platforms;
  - [x] Ran `make fmt` on your commit series before opening this PR;
  - [ ] Updated relevant documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,19 @@
 [![](https://img.shields.io/discord/762976922531528725.svg?label=discord&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)][unikraft-discord]
 [![Go Report Card](https://goreportcard.com/badge/kraftkit.sh)](https://goreportcard.com/report/kraftkit.sh)
 
-Kraftkit is a suite of tools and framework for building custom, minimal, immutable lightweight virtual machines based on [Unikraft](https://unikraft.org).
-With KraftKit, you can use unikernels at every stage of their lifecycle: from construction to production.
+KraftKit provides a suite of tools and Go-based framework for building custom, minimal, immutable lightweight unikernel virtual machines based on [Unikraft](https://unikraft.org): a fast, secure and open-source library Operating System.
+
```
Comment (craciunoiuc) [inline README.md]:
```suggestion
KraftKit provides a suite of tools and Go-based framework for building custom, minimal, immutable lightweight unikernel virtual machines based on [Unikraft](https://unikraft.org): a fast, secure and open-source library operating system.

```

File: README.md [hunk 2]
```diff
@@ -5,8 +5,19 @@
...
+![](docs/demo.gif)
+
+With KraftKit, you can easily leverage Unikraft and manage specialized, high-performance applications at every stage of their lifecycle: from construction to production.
```
Comment (craciunoiuc) [inline README.md:12]:
```suggestion
With KraftKit, you can easily leverage Unikraft and manage bespoke, high-performance applications at every stage of their lifecycle: from construction to production.
```

File: README.md [hunk 3]
```diff
@@ -5,8 +5,19 @@
...
+
+ * [Getting started guide][kraftkit-getting-started] 📖
+ * [Key differences between containers and unikernels](https://unikraft.org/docs/concepts/virtualization/) 🤔
+ * [Join `#kraftkit` on Unikraft's Community Discord](https://bit.ly/UnikraftDiscord) 👾
+
+There are many benefits in running your application as a unikernel: for more information about [the performance of unikernels 🚀](https://unikraft.org/docs/features/performance/), [the added security 🔒](https://unikraft.org/docs/features/security/) and [a positive impact on the environment 🌱](https://unikraft.org/docs/features/green/) please [check out Unikraft's documentation][unikraft-docs] and the introductory chapters on these impacts.
```
Comment (craciunoiuc) [inline README.md:18]:
```suggestion
There are many benefits from running your application as a unikernel: for more information about [the performance of unikernels 🚀](https://unikraft.org/docs/features/performance/), [the added security 🔒](https://unikraft.org/docs/features/security/) and [a positive impact on the environment 🌱](https://unikraft.org/docs/features/green/) please [check out Unikraft's documentation][unikraft-docs] and the introductory chapters on these impacts.
```

**Review by craciunoiuc — CHANGES_REQUESTED** [review]
Some comments from my side

File: README.md (hunk 2 above)
Comment (nderjung) [reply README.md:12]: ha-ha

File: README.md (hunk 3 above)
Comment (nderjung) [reply README.md:18]: I would leave as `in`

**Review by craciunoiuc — APPROVED** [review]
All good here. Thanks!

Reviewed-by: Cezar Craciunoiu <cezar.craciunoiu@unikraft.io>
Approved-by: Cezar Craciunoiu <cezar.craciunoiu@unikraft.io>

**Pull request merged**